README.md

```diff
@@ -58,11 +58,13 @@ Follow the instructions for your distribution:
 
 ## Building from source
 
-Before building, you will need the following
+### Requirements
+
+-	[Terraform](https://www.terraform.io/downloads.html)
```

Comment:
@MalloZup Should I add a line for `golint`, `go get -u golang.org/x/lint/golint`? It is not install when downloading the go dependencies, but it is required to build the project.

Reply:
I think `golint` isn't really a requirement. 

In theory we should move it away from the build target to make it more granular. https://github.com/dmacvicar/terraform-provider-libvirt/blob/master/Makefile#L11

Reply:
we can do it other prs if needed but  I think should be ok without